### DISCUSSION TITLE
[QUESTION] El botón para levantar un formulario ya no aparece el nombre de 1 formulario cuando es 1?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#288] [2. Questions and Answers] [answered]
Hola

Me he dado cuenta que ahora el botón de levantar formularios:
![image](https://user-images.githubusercontent.com/61419471/123124315-92f71480-d415-11eb-81b5-ee53d29f1fb1.png)

Ya no aparece el nombre del formulario disponible cuando es uno solo (por ejemplo)
![image](https://user-images.githubusercontent.com/61419471/123124832-039e3100-d416-11eb-9911-cbca4f4939a8.png)

O ya quitaron esa validación?. Me lo pregunto un cliente.

Gracias =>

**Comment by ccarrascor** [comment, marked as answer]
Ahora siempre aparece el mismo boton. :)

**Reply by guillevalin** [reply]
Así es @NicolasPLG , tal como lo explica @ccarrascor desde el último release ahora el botón "Levantar formulario" pasó a ser el botón de "Acciones", y siempre se levantará un pop-up con todas las acciones disponibles, esto fue pensado también en mejor experiencia de usuario en el caso de formularios o acciones de nombres largos, y homologar la experiencia móvil con la experiencia en escritorio.
